resources/helm/overlays/global.yaml

```diff
@@ -402,15 +402,6 @@ meshConfig:
       ISTIO_META_DNS_AUTO_ALLOCATE: "true"
       # required to enable dns capture
       PROXY_XDS_VIA_AGENT: "true"
-    tracing:
-#      tlsSettings:
-#        mode: DISABLE # DISABLE, SIMPLE, MUTUAL, ISTIO_MUTUAL
-#        clientCertificate: # example: /etc/istio/tracer/cert-chain.pem
-#        privateKey:        # example: /etc/istio/tracer/key.pem
-#        caCertificates:    # example: /etc/istio/tracer/root-cert.pem
-#        sni:               # example: tracer.somedomain
-#        subjectAltNames: []
-        # - tracer.somedomain
```

Comment:
I feel it's better comment out the line 405 and keep the comment lines 406-413 for tlsSettings example references

Reply:
I removed these values, because they are obsolete and no longer used in the charts.